sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -180,7 +180,7 @@ def __repr__(self):
         )[:1024]
 
 
-class USReceipt(object):  # pylint: disable=too-many-instance-attributes
+class USReceipt(RecognizedForm):  # pylint: disable=too-many-instance-attributes
```

Comment:
This is more of a question: If we inherit from another exposed class, do we include the properties we inherited from that other class in our docstring? I did this when I was working on key vault, but just want to check for sure what we should be doing in this case

Reply:
called super() - so it includes the properties

Reply:
we must be doing it imo - since they are "valid" properties. As a customer, i would not want to go check the docsstring for a series of models

Reply:
ok, that makes a lot of sense. thanks!